### DISCUSSION TITLE
how do you put the favicon on my page?

### DISCUSSION THREAD

**Discussion by NarcelioO** [vercel/next.js#22394] [Help]
I researched and tested everything I saw

**Comment by michrome** [comment]
Hey @NarcelioO —

```js
// pages/_app.js
import Head from "next/head";

function MyApp({ Component, pageProps }) {
  return (
    <>
      <Head>
        <link rel="icon" type="image/png" href="/path/to/favicon.png" />
      </Head>
      <Component {...pageProps} />
    </>
  )
}

export default MyApp
```

**Reply by NarcelioO** [reply]
I already did it, it doesn't work at all

**Reply by michrome** [reply]
You can see [the same pattern in the blog-starter example](https://github.com/vercel/next.js/blob/canary/examples/blog-starter/components/meta.js) so I recommend double-checking for typos.

**Reply by agusterodin** [reply]
I wish they had favicon generation out of the box (or some sort of officially sanctioned solution), something similar to gatsby-plugin-manifest. In the meantime here are two solutions:

https://github.com/agusterodin/Modern-Favicons (made by me, very lightweight)
https://github.com/tinialabs/next-favicon-loader (more of a gatsby-plugin-manifest equivalent)
